### DISCUSSION TITLE
Missing Features on SelfHosted?

### DISCUSSION THREAD

**Discussion by MS-Rait** [outline/outline#4885] [Self Hosting] [answered]
Hello,

I was recently browsing the changelogs on the official site and was very happy to see, that two cool new Features have been released i didnt knew about yet:

1. MacOS App
2. Multiple Workspaces

First one is already stated that it is currently only supported on the cloud version. Does anyone know if it is planned to support also the selfhosted version?

But with multiple workspaces there is no indicator that this shouldnt be supported with self hosting it. Am I doing something wrong and it should work or is it not possible to use it whilest selfhosting?

Currently im running 0.67.2

Greets
Marcel

**Comment by MS-Rait** [comment, marked as answer]
Nevermind looking in the other discussions i was able to answer it my self :)
I have searched in the issues but not in the discussions yet.
Sad to know that it is a Enterprise exclusive feature.

**Reply by tommoor** [reply]
I think this year we'll investigate having a paid self-hosted option that isn't enterprise, with a sub-selection of features. Multiple workspaces would be considered for such a license.

**Reply by Limezy** [reply]
This kind of financing model could make a lot of sense

**Reply by kb1900** [reply]
> I think this year we'll investigate having a paid self-hosted option that isn't enterprise, with a sub-selection of features. Multiple workspaces would be considered for such a license.

Yes please! Especially for features like ai integration, maintaining self hosting priced option would be excellent

**Reply by mikaeleiman** [reply]
> I think this year we'll investigate having a paid self-hosted option that isn't enterprise, with a sub-selection of features. Multiple workspaces would be considered for such a license.

In our case I don't think that the current per-user price of Enterprise would be a problem, but the minimum 100 seats in effect triples the price per user since we only have about 30 employees. So a lower seat minimum would be more interesting for us.

**Reply by tommoor** [reply]
The minimum seat count is really to cover the cost of support – if you're already successfully running self-hosted and just want the enterprise features without a support contract then we can provide a license with less seats, just email hello@getoutline.com and we'll get you set up.
